Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <string>
+#include "Person.h"
+using namespace std;
+
+
+
+Person::Person(){
+    age_ = new int(1);
+}
+
+Person::~Person()
+{
+    delete age_;
+}
+
+Person::Person(const std::string &name, int age)
```

Comment:
The ampersand should be on the side of object type: `std::string& name`

Reply:
done